src/Core/Services/Cache/DabCacheService.cs

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
```

Comment:
Something to consider:  We could have an interface like `ICacheService` and have `DabCacheService` implement that. This could be useful if we want to extend to support distributed caching, etc.

Reply:
I did consider that. However, I have not determined whether another implementation of such an `ICacheService` is necessary to accommodate distributed caching versus extending the configuration of `FusionCache` within the original `DabCacheService`. Creating an interface is premature in this case when it is unclear what that interface should enforce in its contract. 
While an interface with only a single implementation could be helpful for testing, I did not want to mock `DabCacheService` in tests because mocking the query executor achieved what i wanted to test: behavior of the cache service as a result of results returned by the query executor.

Reply:
Thanks for the explanation